code/modules/research/techweb/all_nodes.dm

```diff
@@ -486,6 +486,15 @@
 	export_price = 5000
 
 ////////////////////////Medical////////////////////////
+/datum/techweb_node/cyber_organs_upgraded
+	id = "cyber_organs_upgraded"
+	display_name = "Upgraded Cybernetic Organs"
+	description = "We have the technology to upgrade him."
+	prereq_ids = list("cyber_organs")
+	design_ids = list("cybernetic_heart_u", "cybernetic_liver_u", "cybernetic_lungs_u","catcybernetic") //austation - is this too far?, yeah adds cybernetic catears
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+	export_price = 5000
+
```

Comment:
```suggestion
```
No, uh, not like that... You don't actually have to replace the original techweb node to achieve your goals. Apply my suggestion and replace line 530 to ```design_ids = list("cybernetic_heart_u", "cybernetic_liver_u", "cybernetic_lungs_u","catcybernetic") //austation - is this too far?, yeah adds cybernetic catears```

And since this entry is above the original techweb node entry the BYOND probably will try to overwrite yours with the original entry...

Reply:
done nevermind my retardation